src/server.tsx

```diff
@@ -77,7 +77,7 @@ const buildEpic = async (
         abTestName: test.name,
         abTestVariant: variant.name,
         campaignCode: buildCampaignCode(test, variant),
-        campaignId: test.campaignId || test.name,
+        campaignId: `epic_${test.campaignId || test.name}`,
```

Comment:
I wonder why we weren't having to prepend `epic_` [here in DCR](https://github.com/guardian/dotcom-rendering/blob/master/src/web/components/SlotBodyEnd.tsx#L41) to make up for this 🤔 I guess is this a convention rather than having any functional purpose?

Reply:
I think we may want to prepend `epic_` only to the `test.name`, not to `test.campaignId`. But I'm going to try and confirm this with Tom F. As far as I've seen, only hardcoded tests have their own manually defined `campaignId` - configured tests derive this from the test name.

With hardcoded tests the author defines the `campaignId` they want and I've seen one that starts with "epic_" already (all the others I've seen don't). This might not be that important but worth checking I think.

Reply:
Update: double checked Frontend and this logic seems correct. I just thought it was strange to dynamically prepend "epic_" when the user can input whatever value they want, but Frontend does it too and Tom F doesn't think this is an issue.